apps/console/src/features/users/components/guests/pages/guest-users-list.tsx

```diff
@@ -358,11 +364,11 @@ export const GuestUsersList: FunctionComponent<GuestUsersListInterface> = (
     return (
         <>
             <DataTable<UserInviteInterface>
+                className="users-table"
```

Comment:
Do we need to add this class? there are no styles defined..

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/4572/commits/64939d252cbea798171fc1031eacf2ff42fe049c